.grit/patterns/js/graphql-v3-csrf-prevention.md

```diff
@@ -0,0 +1,71 @@
+---
+title: GraphQL Sever v3 csrf prevention
+---
+
+The Apollo GraphQL server lacks the 'csrfPrevention' option. This option is 'false' by the default in v3 of the Apollo GraphQL v3, which can enable CSRF attacks
+
+tags: #fix #graphQL, #security
+
+```grit
+engine marzano(0.1)
+language js
+
+or {
+    `new ApolloServer({$config})` => `new ApolloServer({
+    $config,
+    csrfPrevention: true
+})` where {
+        $config <: not contains `csrfPrevention: $boolean`        
+    },
```

Comment:
```suggestion
    `new ApolloServer({$config})` where {
      $config <: not contains `csrfPrevention: $boolean`,
      $config += `csrfPrevention: true`
    }
```

Reply:
Hi @morgante, updated the suggested changes